README.md

```diff
@@ -856,6 +856,26 @@ python3 edn2json.py input.edn > output.json
 
 ------------------------------------------------------------------------------
 
+## Testing tools
+
+### `st.py` (Stage Tester)
+
+#### Description
+
+This script can be used to retrieve list of clusters from the external data
+pipeline through the standard REST API. Organization ID (a.k.a. account number)
```

Comment:
Org ID != account_number, they're different. They are in a 1:1 relationship, but they're different numbers.